homeassistant/components/hue.py

```diff
@@ -0,0 +1,237 @@
+"""
+homeassistant.components.hue
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for Philips Hue bridge.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/hue/
+"""
+
+import logging
+import socket
+from datetime import timedelta
+from urllib.parse import urlparse
+
+from homeassistant import bootstrap
+from homeassistant.util import Throttle
+from homeassistant.loader import get_component
+from homeassistant.components import discovery
+from homeassistant.const import (
+    CONF_HOST, ATTR_SERVICE, ATTR_DISCOVERED, EVENT_PLATFORM_DISCOVERED)
+
+DOMAIN = 'hue'
+REQUIREMENTS = ['phue==0.8']
+
+_LOGGER = logging.getLogger(__name__)
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=10)
+MIN_TIME_BETWEEN_FORCED_UPDATES = timedelta(milliseconds=100)
+
+PHUE_CONFIG_FILE = "phue.conf"
+
+DISCOVER_LIGHTS = 'hue.light'
+DISCOVER_SWITCHES = 'hue.switch'
+DISCOVER_SENSORS = 'hue.sensor'
+
+# terminology used by phue
+ATTR_HUE_LIGHTS = 'lights'
+ATTR_HUE_BRIDGEID = 'bridgeid'
+ATTR_HUE_BRIDGEIP = 'ipaddress'
+
+# Preparation for tap, switches and dimmers
+DISCOVERY_COMPONENTS = [
+    ('light', DISCOVER_LIGHTS, ATTR_HUE_LIGHTS,),
+    ('switch', DISCOVER_SWITCHES, '',),
+    ('sensor', DISCOVER_SENSORS, '',),
+]
+
+HUEBRIDGE = None
```

Comment:
I find this attribute difficult to read. What about renaming it to `BRIDGES` or `HUE_BRIDGES` ?